pkg/minikube/registry/drvs/hyperv/hyperv.go

```diff
@@ -19,35 +19,57 @@ limitations under the License.
 package hyperv
 
 import (
+	"fmt"
+	"os/exec"
+	"strings"
+
 	"github.com/docker/machine/drivers/hyperv"
 	"github.com/docker/machine/libmachine/drivers"
+
 	cfg "k8s.io/minikube/pkg/minikube/config"
 	"k8s.io/minikube/pkg/minikube/driver"
 	"k8s.io/minikube/pkg/minikube/localpath"
 	"k8s.io/minikube/pkg/minikube/registry"
 )
 
+const (
+	docURL = "https://minikube.sigs.k8s.io/docs/reference/drivers/hyperv/"
+)
+
 func init() {
-	registry.Register(registry.DriverDef{
-		Name:          driver.HyperV,
-		Builtin:       true,
-		ConfigCreator: createHypervHost,
-		DriverCreator: func() drivers.Driver {
-			return hyperv.NewDriver("", "")
-		},
-	})
+	if err := registry.Register(registry.DriverDef{
+		Name:     driver.HyperV,
+		Init:     func() drivers.Driver { return hyperv.NewDriver("", "") },
+		Config:   configure,
+		Status:   status,
+		Priority: registry.Preferred,
+	}); err != nil {
+		panic(fmt.Sprintf("register: %v", err))
+	}
 }
 
-func createHypervHost(config cfg.MachineConfig) interface{} {
+func configure(config cfg.MachineConfig) interface{} {
```

Comment:
I would add a comment for this configure,, does what does it really do ?

and also maybe I would change the config var name to somethign shorter ?